### PR TITLE
rename output

### PR DESCRIPTION
**Pull request by daichengxin**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/quantms/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/quantms _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/preprocess_expdesign.nf [hunk 1]
```diff
@@ -13,17 +13,17 @@ process PREPROCESS_EXPDESIGN {
     path design
 
     output:
-    path "${design.baseName}_design.tsv", emit: ch_expdesign
-    path "${design.baseName}_config.tsv", emit: ch_config
+    path "${design.baseName}_openms_design.tsv", emit: ch_expdesign
+    path "${design.baseName}_openms_config.tsv", emit: ch_config
```
Comment (ypriverol) [inline modules/local/preprocess_expdesign.nf:17]: Why, the dataset needs to keep the design.baseName in `${design.baseName}_openms_config.tsv` and `${design.baseName}_openms_design.tsv` should we removed @jpfeuffer @daichengxin ? leave it something like: 

- openms_design.tsv
- openms_config.tsv

File: modules/local/openms/proteomicslfq/meta.yml [hunk 2]
```diff
@@ -39,6 +39,11 @@ output:
   - out_msstats:
```
Comment (jpfeuffer) [inline modules/local/openms/proteomicslfq/meta.yml:39]: This should also be changed. out_msstats is misleading.

File: modules/local/openms/proteomicslfq/meta.yml [hunk 3]
```diff
@@ -39,6 +39,11 @@ output:
...
       type: file
       description: MSstats file with analysis results for statistical downstream analysis in MSstats.
+      pattern: "*msstats_in.csv"
+  - out_triqler:
```
Comment (jpfeuffer) [inline modules/local/openms/proteomicslfq/meta.yml:43]: triqler_in. with corresponding changes in pmultiqc

File: nextflow_schema.json [hunk 4]
```diff
@@ -743,6 +743,12 @@
                     "default": "unique_peptides",
                     "enum": ["unique_peptides", "strictly_unique_peptides", "shared_peptides"],
                     "fa_icon": "fas fa-list-ol"
+                },
+                "export_mztab": {
+                    "type": "boolean",
+                    "description": "Export the results with mzTab format.",
```
Comment (jpfeuffer) [inline nextflow_schema.json:749]:
```suggestion
                    "description": "Export the results in mzTab format.",
```

**Comment by ypriverol** [comment]
@daichengxin can you changed to:

`{design.baseName}_msstats_in.csv`
`${design.baseName}_msstats_out.csv`
`${design.baseName}_triqler_in.tsv`
`${design.baseName}_openms_design.tsv`
`${design.baseName}_config.tsv`

**Review by ypriverol — APPROVED** [review]

**Pull request merged**